sharding-core/sharding-core-parse/sharding-core-parse-mysql/src/main/antlr4/imports/mysql/BaseRule.g4

```diff
@@ -20,27 +20,45 @@ grammar BaseRule;
 import Keyword, DataType, Symbol;
 
 ID 
-    : (BQ_?[a-zA-Z_$][a-zA-Z0-9_$]* BQ_? DOT_)? (BQ_?[a-zA-Z_$][a-zA-Z0-9_$]* BQ_?)
+    : [A-Za-z_$0-9]*?[A-Za-z_$]+?[A-Za-z_$0-9]*
+    ;
+    
+REVERSE_QUOTE_ID
+    : '`' ~'`'+ '`'
+    ;
+    
+uid
```

Comment:
what's mean of uid? should use more meaningful name here

Reply:
Unique Id,  i think it is ok!

Reply:
Please avoid use abbreviation. Another question is what is Unique Id in SQL?